os/arch/arm/src/common/up_internal.h

```diff
@@ -269,6 +269,7 @@ EXTERN uint32_t _sdata;			/* Start of .data */
 EXTERN uint32_t _edata;			/* End+1 of .data */
 EXTERN uint32_t _sbss;			/* Start of .bss */
 EXTERN uint32_t _ebss;			/* End+1 of .bss */
+EXTERN uint32_t _sidle_stack;		/* Start of idle stack */
```

Comment:
How about adding the variables for internal and external heap also here?

Reply:
Correct, updated them